doc/user/layouts/shortcodes/views-indexes/index-considerations.html

```diff
@@ -0,0 +1,13 @@
+Before creating an index, consider the following:
+
+- If you have created various views to act as the building blocks for a view
+  from which you will serve the results, creating an index **only** on the
+  serving view may be sufficient depending upon your query patterns and usage.
+
+- In Materialize, underlying data structures that are being maintained for an
+  index (or a materialized view) can be reused by other views/queries. As such,
+  an existing index may be sufficient to support your queries.
+
+- For queries that have no supporting indexes, Materialize uses the same
+  mechanics used by indexes to optimize computations. However, this underlying
+  work is discarded after each query run.
```

Comment:
What point are you trying to make here? It might be more confusing than helpful.

Reply:
Ah, so, this was mainly to prevent people from indexing everything.